### ISSUE TITLE
[BUG] Preview doesn't work as expected

### ISSUE THREAD

**Issue by orkenstein** [fermoya/SwiftUIPager#76]
**Describe the bug**
Canvas preview in Xcode does not render the pager.

**To Reproduce**
```
import SwiftUI
import SwiftUIPager

struct WelcomeView: View {
  @State var page: Int = 0
  var items = Array(0..<10)

  var body: some View {
    Pager(page: $page,
          data: items,
          id: \.self,
          content: { index in
            // create a page based on the data passed
            Text("Page: \(index)").background(Color.yellow).frame(width: 100, height: 100)
    }).background(Color.green)
  }
}

struct WelcomeView_Previews: PreviewProvider {
  static var previews: some View {
    WelcomeView()
  }
}
```
**Expected behavior**
Expected to have rendered view in preview.

**Screenshots / Videos**
<img width="400" alt="Screen Shot 2020-07-21 at 16 42 36" src="https://user-images.githubusercontent.com/1461936/88062234-3ccc8200-cb71-11ea-99e5-aea24986f366.png">

**Environment:**
 - xcode 11.6 (11E708)

**Comment by fermoya** [comment]
Hi @orkenstein , I’m well aware of this issue. It’s due to the dependence of `Pager` on `SizeModifier`. It makes it not run the Preview but makes the code more readable and maintainable.

I’m not going to prioritize this as I there are other issues that are more urgent. Nonetheless, I’ll be taking a look at this promptly and see if there’s anything I could improve there to make it work.

Thanks for your feedback

**Comment by fermoya** [comment]
Fixed in version [1.9.0](https://github.com/fermoya/SwiftUIPager/releases/tag/1.9.0). See preview example [here](https://github.com/fermoya/SwiftUIPager/blob/51c81003d87bea61a171f4f5dd3cbb3f719c9941/Example/SwiftUIPagerExample/Examples/InfiniteExampleView.swift#L86).
<img width="1792" alt="Screenshot 2020-07-24 at 18 13 52" src="https://user-images.githubusercontent.com/11335612/88417231-7e5f6600-cdd9-11ea-8785-a7ece2108d50.png">

**Comment by orkenstein** [comment]
Yes, works just fine, thanks a lot! 🔥
